<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>andor_newton_config</class>
 <widget class="QWidget" name="andor_newton_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" rowspan="3">
    <widget class="QFrame" name="tracks_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3"/>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_5">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_15">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Temperature (°C)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="set_temperature">
        <property name="minimum">
         <number>-99</number>
        </property>
        <property name="value">
         <number>-60</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit_current_temperature">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="4" column="0">
       <widget class="QSpinBox" name="preamp_gain">
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Readout Mode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="comboBox_readout_mode">
        <item>
         <property name="text">
          <string>FVB - full vertical binning</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>multi track</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>image</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>random track</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Preamp Gain</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Readout</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QSpinBox" name="horizontal_binning">
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QDoubleSpinBox" name="hs_speed">
        <property name="value">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>HS Speed (MHz)</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QDoubleSpinBox" name="vs_speed">
        <property name="value">
         <double>25.699999999999999</double>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>VS Speed (µs/shift)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Horizontal Binning</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QComboBox" name="comboBox_shutter_mode">
        <item>
         <property name="text">
          <string>open</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>closed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>auto</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QDoubleSpinBox" name="exposure_time">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Exposure</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Exposure Time (s)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Shutter Mode</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Shutter TTL open</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QComboBox" name="comboBox_shutter_ttl">
        <item>
         <property name="text">
          <string>low</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>high</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>set_temperature</tabstop>
  <tabstop>exposure_time</tabstop>
  <tabstop>preamp_gain</tabstop>
  <tabstop>horizontal_binning</tabstop>
  <tabstop>hs_speed</tabstop>
  <tabstop>vs_speed</tabstop>
  <tabstop>comboBox_shutter_mode</tabstop>
  <tabstop>comboBox_readout_mode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
